gradle/dist.gradle

```diff
@@ -0,0 +1,89 @@
+import java.nio.file.Paths
+
+import static groovy.io.FileType.FILES
+
+def licenses = [
+        [name: 'avalon-framework', title: 'Avalon Framework', license: ['apache2.txt']],
+        [name: 'batik', title: 'Batik', license: ['apache2.txt']],
+        [name: 'xmlgraphics-commons', title: 'Apache XML Graphics Commons', license: ['apache2.txt']],
+        [name: 'commons-io', title: 'Commons IO', license: ['apache2.txt']],
+        [name: 'xercesImpl', title: 'Xerces2 Java', license: ['apache2.txt']],
+        [name: 'xml-apis', title: 'XML Standard API', license: ['apache2.txt']],
+        [name: 'xml-resolver', title: 'Resolver', license: ['apache2.txt']],
+        [name: 'Saxon-HE', title: 'Saxon-HE', license: ['mpl2.txt', 'CERN.txt', 'JAMESCLARK.txt', 'THAI.txt', 'UNICODE.txt']],
+        [name: 'icu4j', title: 'ICU4J', license: ['icu.txt']],
+        [name: 'ant', title: 'Ant', license: ['apache2.txt']],
+        [name: 'guava', title: 'Guava', license: ['apache2.txt']],
+        [name: 'slf4j', title: 'SLF4J', license: ['mit-slf4j.txt']],
+        [name: 'jcl-over-slf4j', title: 'SLF4J', license: ['mit-slf4j.txt']],
+        [name: 'logback', title: 'Logback', license: ['epl-1.0.txt']],
+        [name: 'jackson', title: 'Jackson', license: ['apache2.txt']],
+        [name: 'jing', title: 'Jing', license: ['bsd-jing.txt']],
+        [name: 'gradle-wrapper'],
```

Comment:
These license entries without `licence` are markers for our own JAR files, so we know to skip them.